File: Python/payment/Card.py
from payment.Payment import Payment
class Card(Payment):
    __number = str
    __cvv = str
    __date = str
    def __init__(self,number,cvv,date):
        self.__number = number
        self.__cvv = cvv
        self.__date = date

    def setNumber(self,number):
        self.__number=number
    
    def getNumber(self):
        return self.__number

    def setCvv(self,cvv):
        self.__cvv=cvv
    
    def getCvv(self):
        return self.__cvv

    def setDate(self,date):
        self.__date=date
    
    def getDate(self):
        return self.__date

File: PHP/payment/Card.php
<?php
require_once('Payment.php');
class Card extends Payment{
    private $number;
    private $cvv;
    private $date;

    public function __construct($number,$cvv,$date)
    {
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
    public function setNumber($number){
        $this->number = $number;
    }
    public function setCvv($cvv){
        $this->cvv = $cvv;
    }
    public function setDate($date){
        $this->date = $date;
    }
    public function getNumber(){
        return $this->number;
    }
    public function getCvv(){
        return $this->cvv;
    }
    public function getDate(){
        return $this->date;
    }
}
?>